spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -46,83 +46,79 @@ MicroProfile applications that make use of APIs or versions not mentioned below
-Use of implementations beyond JAX-RS 2.1 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+JAX-RS provides both standard client and server APIs for RESTful communication by MicroProfile 4.0 applications.
+Use of implementations beyond JAX-RS 2.1 in MicroProfile is allowed, however, not required as of version 4.0 of MicroProfile.
 
- - https://github.com/jax-rs
- - https://jcp.org/en/jsr/detail?id=370
+ - https://jakarta.ee/specifications/restful-ws/2.1/restful-ws-spec-2.1.html
+ - https://github.com/eclipse-ee4j/jaxrs-api
 
-[[javaee-jsonb]]
-=== JSON-B 1.0
+[[jakarta-jsonb]]
+=== Jakarta JSON-B 1.0
 
-JSON-B is leveraged as part of MicroProfile 2.0 to provide standard APIs for binding JSON documents to Java code.
-Use of implementations beyond JSON-B 1.0 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+JSON-B is leveraged as part of MicroProfile 4.0 to provide standard APIs for binding JSON documents to Java code.
+Use of implementations beyond JSON-B 1.0 in MicroProfile is allowed, however, not required.
 
- - https://javaee.github.io/jsonb-spec
- - https://jcp.org/en/jsr/detail?id=367
+ - https://jakarta.ee/specifications/jsonb/1.0/jsonb-spec-1.0.html
+ - https://github.com/eclipse-ee4j/jsonb-api
 
-[[javaee-jsonp]]
-=== JSON-P 1.1
+[[jakarta-jsonp]]
+=== Jakarta JSON-P 1.1
 
-JSON-P is leveraged as part of MicroProfile 2.0 to provide standard APIs for processing JSON documents.
-Use of implementations beyond JSON-P 1.1 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+JSON-P is leveraged as part of MicroProfile 4.0 to provide standard APIs for processing JSON documents.
+Use of implementations beyond JSON-P 1.1 in MicroProfile is allowed, however, not required.
 
- - https://javaee.github.io/jsonp/
- - https://jcp.org/en/jsr/detail?id=374
+ - https://jakarta.ee/specifications/jsonp/1.1/jsonp-spec-1.1.html
+ - https://github.com/eclipse-ee4j/jsonp
 
-[[javaee-common-annotations]]
-=== Common Annotations 1.3
+[[jakarta-annotations]]
+=== Jakarta Annotations 1.3
 
-Common Annotations provides annotations for common semantic concepts across a variety of individual technologies in the Java(TM) SE and Java(TM) EE platforms.
-Use of implementations beyond Common Annotations 1.3 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+Jakarta Annotations provides annotations for common semantic concepts across a variety of individual technologies in the Java(TM) SE and Jakarta EE platforms.
+Use of implementations beyond Jakarta Annotations 1.3 in MicroProfile is allowed, however, not required.
 
- - https://javaee.github.io/glassfish/
- - https://jcp.org/en/jsr/detail?id=250
+ - https://jakarta.ee/specifications/annotations/1.3/annotations-spec-1.3.html
+ - https://github.com/eclipse-ee4j/common-annotations-api
 
 [[mp-config]]
-=== Eclipse MicroProfile Config 1.4
+=== Eclipse MicroProfile Config 2.0
```

Comment:
All of these MP releases are Milestone or Release Candidate releases.  Shouldn't we be indicating that these are not final releases?

Reply:
Ok, this is a bit of admitted double-speak for me. I provide the milestone release links in the architecture doc but intentionally point to "final" releases in the required-apis doc.  I've done this in the past as well. I can fix this but hoping to avoid it for less work later. Fix anyway?

Reply:
sorry, "double-speak **from me**"

Reply:
I can understand about not making extra work for the RCx releases, since we're so close to actually releasing something when we get to that stage.  But, the Mx releases are different.  We don't have a timeframe for when the next Milestone or RC or final release of 4.0 will happen...  These Mx releases will be a bit more work (and they should be).

Reply:
Will fix.